config/ember-try.js

```diff
@@ -24,6 +24,7 @@ module.exports = function() {
           },
           npm: {
             devDependencies: {
+              'ember-cli-htmlbars-inline-precompile': '~1.0.5',
```

Comment:
I think this needs to be moved to `dependencies` (not `devDependencies`), otherwise we would have `dependencies` of `2.0.0` and `devDependencies` of `1.0.5` and my guess is that `yarn` chooses `dependencies` to "win" over dev deps.

Also can you change these to `^1.0.5`